### PR TITLE
Hidden attributes and stateless transactions

### PR DESCRIPTION
**Pull request by MarkvanMents**
[TW-1522] raised by Rik Bos

### CODE REVIEW

File: content/refguide/access-rules.md [hunk 1]
```diff
@@ -94,6 +94,10 @@ For example, a customer is allowed to view the discount, but is not allowed to e
 
 ![](attachments/domain-model/access-rule-discount-read.png)
 
+{{% alert type="warning" %}}
+If a user cannot view the value of an attribute because of security constraints, that attribute will never be sent to the Mendix Client. Because Mendix is stateless, this can lead to unexpected results (for example, loss of changes) if changes to the attribute in a microflow are not committed immediately. See [Basic CRUD Communication Pattern](communication-patterns#crud) in *Communication Patterns in the Mendix Runtime* for more information on how data is passed between the Runtime Server and the Mendix Client.
```
Comment (rik-bos) [inline content/refguide/access-rules.md:98]: I'd explicitly add that there will be more explanation about this case. E.g.:

```[...] for more information on how data is passed between the Runtime Server and the Mendix Client and what cases may lead to loss of changes.```

Comment (MarkvanMents) [reply content/refguide/access-rules.md]: Thanks' @rik-bos 
I've made that change.
Are you OK with me publishing now?
Thanks

Comment (rik-bos) [reply content/refguide/access-rules.md]: @MarkvanMents : looks great!

**Comment by MarkvanMents** [comment]
Publishing

**Pull request merged**